config/api_client.go

```diff
@@ -73,17 +75,22 @@ func (c *Config) NewApiClient() (*httpclient.ApiClient, error) {
 				return nil
 			},
 		},
-		TransientErrors: []string{
-			"REQUEST_LIMIT_EXCEEDED", // This is temporary workaround for SCIM API returning 500.  Remove when it's fixed
-		},
 		ErrorMapper: apierr.GetAPIError,
-		ErrorRetriable: func(ctx context.Context, err error) bool {
-			var apiErr *apierr.APIError
-			if errors.As(err, &apiErr) {
-				return apiErr.IsRetriable(ctx)
-			}
-			return false
-		},
+		ErrorRetriable: httpclient.CombineRetriers(
+			func(ctx context.Context, _ *http.Request, _ *common.ResponseWrapper, err error) bool {
+				var apiErr *apierr.APIError
+				if errors.As(err, &apiErr) {
+					return apiErr.IsRetriable(ctx)
+				}
+				return false
+			},
+			httpclient.RetryUrlErrors,
```

Comment:
I moved this out of the ApiClient to have a single codesite where the retry logic is defined for the client. The downside is that you need to add this explicitly in your ErrorRetriable if you don't specify DefaultErrorRetriable. Happy to make this a default behavior, let me know what you think.